### PR TITLE
Infra Env wizard

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: src/components/Infra/InfraEnvDetailsPage.tsx [hunk 1]
```diff
@@ -3,22 +3,30 @@ import { match as RMatch } from 'react-router-dom';
...
+  name: string;
 }
 
-const InfraEnvDetails: React.FC<InfraEnvDetailsProps> = (props) => {
+const InfraEnvDetails: React.FC<InfraEnvDetailsProps> = ({ namespace, name, ...rest }) => {
   return (
     <DetailsPage
-      {...props}
+      {...rest}
       kind="agent-install.openshift.io~v1beta1~InfraEnv"
```
Comment (jtomasek) [inline src/components/Infra/InfraEnvDetailsPage.tsx:18]:
```suggestion
      kind={InfraEnvKind}
```

File: src/components/Infra/InfraEnvWizard.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    setName(values.name);
+    await k8sCreate(secretModel, {
+      kind: 'Secret',
+      apiVersion: 'v1',
+      metadata: {
+        name: values.name,
+        namespace: 'assisted-installer'
+      },
+      data: {
+        '.dockerconfigjson': Base64.encode(values.pullSecret),
```
Comment (mareklibra) [inline src/components/Infra/InfraEnvWizard.tsx:26]: Why don't we use simple `atob` or `btoa`?

File: src/components/Infra/InfraEnvWizard.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
+import * as React from 'react'
+import { InfraWizard } from 'openshift-assisted-ui-lib';
+import { useK8sModel, k8sCreate, history, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk/api';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+import { load } from 'js-yaml';
+
+import { InfraEnvKind } from '../../kind';
+
+import 'openshift-assisted-ui-lib/dist/index.css';
```
Comment (mareklibra) [inline src/components/Infra/InfraEnvWizard.tsx:9]: This is not needed, already included in the `styles.scss`

Comment (rawagner) [reply src/components/Infra/InfraEnvWizard.tsx]: changed the import to `../styles.scss`;

File: src/components/Infra/InfraEnvWizard.tsx (hunk 2 above)
Comment (rawagner) [reply src/components/Infra/InfraEnvWizard.tsx]: no reason, `btoa` it is!

File: src/components/Infra/InfraEnvDetailsPage.tsx (hunk 1 above)
Comment (rawagner) [reply src/components/Infra/InfraEnvDetailsPage.tsx]: updated

**Review by mareklibra — APPROVED** [review]
The https://github.com/openshift-assisted/assisted-ui-lib/pull/605 is merged now.

**Pull request merged**